### PR TITLE
Added fpValueUpdated

### PR DESCRIPTION
**Pull request by justin-wong-ce**
Added notification to stack on each increment of analogValue with fpValueUpdated. This occurs in the main loop.

### CODE REVIEW

File: BACnetServerExample.py [hunk 1]
```diff
@@ -84,386 +85,738 @@
...
-        "BACnetIPUDPPort": 47808 
-    }
+        "instance": 50,
+        "objectName": "NetworkPort Vermillion",
+        "BACnetIPUDPPort": 47808,
+        "ipLength": 4,
+        "ipAddress": [192, 168, 1, 199],
+        "ipDefaultGateway": [192, 168, 1, 99],
+        "ipDnsServer": [1, 2, 3, 4, 5],
+        "ipNumOfDns": 1,
```
Comment (funvill) [inline BACnetServerExample.py]: This should be computed based on the size of ipDnsServer

File: BACnetServerExample.py [hunk 2]
```diff
@@ -84,386 +85,738 @@
...
+            return True
+    elif propertyIdentifier == bacnet_propertyIdentifier["ipsubnetmask"]:
+        if objectType == bacnet_objectType["networkPort"] and objectInstance == db["networkPort"]["instance"]:
+            valueElementCount[0] = db["networkPort"]["ipLength"]
+            octetStringCopy(db["networkPort"]["ipSubnetMask"], value, valueElementCount[0])
+            return True
+    elif propertyIdentifier == bacnet_propertyIdentifier["ipdnsserver"]:
+        if objectType == bacnet_objectType["networkPort"] and objectInstance == db["networkPort"]["instance"]:
+            valueElementCount[0] = db["networkPort"]["ipNumOfDns"] * 4
+            for i in range(db["networkPort"]["ipNumOfDns"]):
```
Comment (funvill) [inline BACnetServerExample.py:358]: Take the size of db["networkPort"]["ipDnsServer"] and use that to compute the ipNumOfDns

**Comment by funvill** [comment]
In your source code prettify function set the line length to 100 instead of 80. This will help reduce the number of lines that are line feed wrapped.

**Comment by justin-wong-ce** [comment]
I agree it feels very wrapped especially with the way `db` is setup, but it turns out my project was already set at 120 before. I changed my prettifier line length to 150 now.

**Pull request merged**